pkg/cmd/push/cmd.go

```diff
@@ -118,7 +118,7 @@ func runCommand(opts *pushOptions) func(*cobra.Command, []string) error {
 		output.Infof("Pushing %s", opts.modelRef.String())
 		desc, err := PushModel(cmd.Context(), localRepo, remoteRepo, opts)
 		if err != nil {
-			return output.Fatalf("Failed to push: %s", err)
+			return output.Fatalf("Failed to push: %s. Ensure that the organization and repository you are pushing to exist and that you have the necessary write permissions.", err)
```

Comment:
Change
"Ensure that the organization and repository you are pushing to exist and that you have the necessary write permissions."

to
"Check you have write permissions to the organization and repository you are pushing to."